service_management/conf/routes

```diff
@@ -26,3 +26,7 @@ GET         /:mid/samples                                    controllers.analysi
 PUT         /:mid/samples/:id                                controllers.analysis.SampleObjectController.update(mid: Int, id: String)
 GET         /:mid/samples/:id                                controllers.analysis.SampleObjectController.getById(mid: Int, id: String)
 GET         /:mid/samples/:parentId/children                 controllers.analysis.SampleObjectController.getForParentObject(mid: Int, parentId: String)
+
+# Loan endpoints
+POST        /:mid/loans                                      controllers.loan.LoanController.createLoan(mid: Int)
```

Comment:
Might be, but I didn't want to create to many endpoint before we know how it will be used.